twirp.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.required_ruby_version = '>= 1.9'
-  spec.add_runtime_dependency 'google-protobuf', '~> 3.0', '>= 3.7.0'
```

Comment:
wouldnt this only allow 4.x versions? and not 3.x anymore?

Reply:
Fair point. I have updated the PR to allow for this.